OpenRA.Mods.TS/UtilityCommands/LegacySequenceImporter.cs

```diff
@@ -0,0 +1,197 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2015 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation. For more information,
+ * see COPYING.
+ */
+#endregion
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using OpenRA.Mods.Common.FileFormats;
+
+namespace OpenRA.Mods.TS.UtilityCommands
+{
+	class ImportLegacySequenceCommand : IUtilityCommand
+	{
+		public bool ValidateArguments(string[] args)
+		{
+			return args.Length >= 2;
+		}
+
+		public string Name { get { return "--sequence-import"; } }
+
+		IniFile file;
+
+		[Desc("FILENAME", "Convert ART.INI to the OpenRA sequence definition format.")]
+		public void Run(ModData modData, string[] args)
+		{
+			// HACK: The engine code assumes that Game.modData is set.
+			Game.ModData = modData;
+
+			file = new IniFile(File.Open(args[1], FileMode.Open));
+
+			foreach (var section in file.Sections)
+			{
+				var sequence = section.GetValue("Sequence", string.Empty);
+				if (!string.IsNullOrEmpty(sequence))
+					ImportInfantrySequences(section, sequence);
+
+				var foundation = section.GetValue("Foundation", string.Empty);
+				if (!string.IsNullOrEmpty(foundation))
+					ImportBuildingSequences(section);
+			}
+		}
+
+		void ImportBuildingSequences(IniSection section)
+		{
+			Console.WriteLine(section.Name + ":");
+
+			Console.WriteLine("\tDefaults:");
```

Comment:
No tileset codes/extensions is quite the exception for TS/RA2 mods. These auto-generated rules need a cleanup afterwards anyway.